loops.js

```diff
@@ -33,7 +43,14 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
-
+for (let i = 0; i<16; i++){
+    if (i % 2 === 0){
+        console.log((i) + ' is even!')
+    }
+    if (i % 2 === 1){
+        console.log((i) + ' is oneven!')
+    }
+}
 
 // ==========================================
```

Comment:
for (let i = 0; i < 16; i++) {
    if (i % 2 == 0) {
        console.log(i + " - even")
    } else {
        console.log(i + " - oneven")
    }
}

dit is mijn versie, ook weer met 'else'